1-exercises/B-while-loop/exercise.js

```diff
@@ -7,7 +7,14 @@
 
 function evenNumbers(n) {
     // TODO
+    i = 0;
+    let even = [];
+    while (i < 2 * n){
+    even.push(i);}
+    i += 2
```

Comment:
Think your syntax is off here; you're incrementing `i` _after_ the while loop is done.

Also, for checking even numbers, the modulo `%` operator is what you're looking for.